pkg/controllers/mcs/service_import_controller.go

```diff
@@ -124,17 +125,23 @@ func (c *ServiceImportController) updateServiceStatus(svcImport *mcsv1alpha1.Ser
 			IP: ip,
 		})
 	}
-	derivedService.Status = corev1.ServiceStatus{
-		LoadBalancer: corev1.LoadBalancerStatus{
-			Ingress: ingress,
-		},
-	}
 
-	if err := c.Client.Status().Update(context.TODO(), derivedService); err != nil {
+	err := retry.RetryOnConflict(retry.DefaultBackoff, func() (err error) {
+		if err = c.Get(context.TODO(), client.ObjectKey{Namespace: derivedService.Namespace, Name: derivedService.Name}, derivedService); err != nil {
+			return err
+		}
```

Comment:
I just realized this will introduce an extra `Get` operation when there is no conflict, right?

Reply:
yes,get object again before patching status